tests/HasTagsScopesTest.php

```diff
@@ -115,4 +122,36 @@ public function it_provides_as_scope_to_get_all_models_that_have_all_of_the_give
 
         $this->assertEquals(['model5'], $testModels->pluck('name')->toArray());
     }
+
+    /** @test */
+    public function it_provides_as_scope_to_get_all_models_that_have_all_of_the_given_tags_with_type_and_a_specific_locale()
+    {
+        $testModels = TestModel::withAllTags(['tagG'], 'translatedTag' ,'fr')->get();
+
+        $this->assertEquals(['model7'], $testModels->pluck('name')->toArray());
```

Comment:
Make this more clear by seeding tags in two different locales and assets that for each locale you get the right string back.

Reply:
Fixed, thank you